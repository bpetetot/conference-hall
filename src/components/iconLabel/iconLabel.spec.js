/* eslint-env jest */
import snap from 'tests/snapshot'
import IconLabel from './index.js'

const snapshot = (props) => snap(IconLabel)({ ...props })

describe('components/iconLabel', () => {
  it('should not render if no label', snapshot({ icon: 'i' }))
  it('should render icon label', snapshot({ icon: 'i', label: 'l' }))
  it('should render icon label with class', snapshot({ icon: 'i', label: 'l', className: 'c' }))
  it('should render icon label on right', snapshot({ icon: 'i', label: 'l', right: true }))
})
