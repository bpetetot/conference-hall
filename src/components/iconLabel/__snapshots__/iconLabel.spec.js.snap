// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/iconLabel should not render if no label 1`] = `null`;

exports[`components/iconLabel should render icon label 1`] = `
<span
  className="icon-label"
>
  <span>
    <i
      className="i"
    />
  </span>
  <span
    className=""
  >
    l
  </span>
</span>
`;

exports[`components/iconLabel should render icon label on right 1`] = `
<span
  className="icon-label icon-label-right"
>
  <span
    className=""
  >
    l
  </span>
  <span>
    <i
      className="i"
    />
  </span>
</span>
`;

exports[`components/iconLabel should render icon label with class 1`] = `
<span
  className="c icon-label"
>
  <span>
    <i
      className="i"
    />
  </span>
  <span
    className=""
  >
    l
  </span>
</span>
`;
